### PR TITLE
draft-ietf-ccamp-optical-impairment-topology-yang.xml and -10.txt for IETF 114 submission

### PR DESCRIPTION
**Pull request by dieterbeller**
draft-ietf-ccamp-optical-impairment-topology-yang.xml for IETF 114 submission --> draft-ietf-ccamp-optical-impairment-topology-yang-10.txt

Related to pull request #111.
Addressing and resolving several open issues.

### CODE REVIEW

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml [hunk 1]
```diff
@@ -1541,6 +1563,68 @@ Line |     .  +-----+ | |  *  *  | | +-----+  .     | Line
...
 
+  <section title="Protection Architectures" anchor="Prot">
+  
+  <t>
+   The YANG model defined in this document supports the following
+   protection architectures:
+  </t>
+    
+  <t><list style="symbols"><?rfc subcompact="yes"?>
+   <t>End-to-end OTSi protection</t>
```
Comment (italobusi) [inline I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:1574]: If the OT is external, this mechanism is modelled as a segmented protection

I would just call this as "OTSi protection" or re-use the definition in Appendix III/G:873.1 and call it "Individual OTSi(G) protection"

**Review by italobusi — APPROVED** [review]

**Review by italobusi — APPROVED** [review]
May need to rebase this branch with the master branch to make sure all the validation checks pass

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml (hunk 1 above)
Comment (sergiobelotti) [reply I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:1574]: Good catch Italo: as depicted in slide 13-14 our model of protection is starting from the port of the splitter on WDM-TE node hosting also the ROADM. So it is a segment protection.

Comment (dieterbeller) [reply I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:1574]: I incorporated the comment and will do another commit.

**Review by sergiobelotti — APPROVED** [review]

**Pull request merged**